pkg/cri/annotations/annotations.go

```diff
@@ -64,6 +64,9 @@ const (
 	// SandboxNamespace is the name of the namespace of the sandbox (pod)
 	SandboxNamespace = "io.kubernetes.cri.sandbox-namespace"
 
+	// ContainerCgroupNamespace is the sandbox annonation for running container in new cgroup namespace.
+	ContainerDisableCgroupNamespace = "io.kubernetes.cri.disable-container-cgroupns"
+
```

Comment:
I guess no label should be added within this PR

Reply:
Yes, I agree with you. Whether to add this `annonation` may need to be reconsidered.